CONTRIBUTING.md

```diff
@@ -10,10 +10,20 @@ We use GitHub to track issues and feature requests, as well as accept pull reque
 You are welcome to contribute with your pull requests. These steps explain the contribution process:
 
 1. Fork the repository and create your branch from `master`.
-2. If you've added code that should be tested, add tests.
-3. If you've changed APIs, update the documentation.
-4. Make sure the tests pass.
-5. Sign the Developer Certificate of Origin (DCO).
+2. Run `npm install` in the root folder. This will install pre-commit and pre-push hooks that will be activated on your pull request.
+3. [Add tests](#testing) for your code, especially if you are updating the Luigi Core API.
```

Comment:
```suggestion
2. Run `npm install` in the root folder. This will install pre-commit and pre-push hooks that will take care about certain required preconditions.
3. [Add tests](#testing) for your code, especially if you are updating the Luigi Core and Client API.
```